src/wled2.ts

```diff
@@ -134,6 +136,29 @@ export = (RED: Red): void => {
     this.send(msg);
   }
 
+  async function getEffectFromString(address: string, payloadEffect: string | number | null, configEffect: string | number | null) {

```

Comment:
This duplicates much of the code from src/controllers/effects.ts. Can it just use the effects() function from there as the base and then do a string lookup on the resulting array instead? If neccessary the meat of effects.ts could be refactored into a common method elsewhere and then just called from both places.

Instead of using `string | number | null` all over the place you can create a new type called WLEDEffect or something and define it to be `string | number | null`. Then all these places wind up being `payloadEffect: WLEDNumber`.